File: DotNet/CesiumLanguageWriter/Advanced/ICesiumSphericalListValuePropertyWriter.cs
using System.Collections.Generic;

namespace CesiumLanguageWriter.Advanced
{
    /// <summary>
    /// A writer that can write a value as a list of spherical values.
    /// </summary>
    public interface ICesiumSphericalListValuePropertyWriter : ICesiumPropertyWriter
    {
        /// <summary>
        /// Writes the value as a list of spherical values.
        /// </summary>
        /// <param name="values">The values.</param>
        void WriteSpherical(IEnumerable<Spherical> values);
    }
}

File: Java/CesiumLanguageWriter/translatedSrc/cesiumlanguagewriter/advanced/ICesiumSphericalListValuePropertyWriter.java
package cesiumlanguagewriter.advanced;


import agi.foundation.compatibility.*;
import cesiumlanguagewriter.*;

/**
 * A writer that can write a value as a list of spherical values.
 */
@SuppressWarnings({
    "unused",
    "deprecation",
    "serial"
})
public interface ICesiumSphericalListValuePropertyWriter extends ICesiumPropertyWriter {
    /**
    * Writes the value as a list of spherical values.
    * @param values The values.
    */
    void writeSpherical(Iterable<Spherical> values);
}